pkg/workflow/controllers/chaos_node_reconciler.go

```diff
@@ -120,10 +122,12 @@ func (it *ChaosNodeReconciler) injectChaos(ctx context.Context, node v1alpha1.Wo
 	}
 
 	updateError := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		namespace := node.Namespace
+		name := node.Name
 		node := v1alpha1.WorkflowNode{}
```

Comment:
how about renaming the `node`?  It's easy to confuse it with the `node` out of this function 😢

Reply:
sure!

Reply:
fixed in b80e9d2